_posts/2024-09-17-pyopensci-sustainability-plan-2024.md

```diff
@@ -0,0 +1,171 @@
+
+Today, the results of this momentum are clear. pyOpenSci has over **250 contributors** and runs a vibrant global Python software peer review process led by [**18 volunteer editors**](https://www.pyopensci.org/about-peer-review/index.html#meet-our-editorial-board), 74 reviewers, and 78 open source maintainers who live around the world.
+
+### Funding supports beginner-friendly open source sprints
+
+Funding has also allowed us to run events. In the past year, over 51 people have participated in our beginner-friendly sprints, submitting 39 issues and 88 pull requests to pyOpenSci. Many of these pull requests and issues are first-time contributions from folks who were both excited about and nervous about contributing to an open source repository.
+
+Contribution to pyOpenSci overall has also sky-rocketed in the past 2 years.
+
+<figure>
+    <a href="{{ site.baseurl }}/images/open-source/pyopensci-contributions-by-quarter.png"><img src="{{ site.baseurl }}/images/open-source/pyopensci-contributions-by-quarter.png" alt="Bar chart titled 'Staff vs. Volunteer Contributor by Quarter.' The chart shows counts of contributions made by staff and volunteers from 2018 Q4 through 2024 Q3. Contributions are stacked, with purple representing staff and green representing volunteers. The chart reveals a significant growth in contributions over time, particularly from 2022 onward, with a notable increase in both staff and volunteer contributions in 2023 and 2024. The pyOpenSci logo is in the bottom-right corner."></a>
+    <figcaption>A bar plot showing staff vs volunteer contributions on GitHub over time. Funding has allowed us to support more volunteer activity over the past 2 years.
+</figcaption>
+</figure>
+
+
+### Our Python packaging guide is making a splash
+
+One of our most impactful activities has been the development of our community-vetted Python packaging guide which includes beginning-to-end Python packaging tutorials using Hatch. This guidebook was developed through a truly collaborative effort, where we built consensus around critical packaging decisions that come with numerous opinions around what is "best." In just under a year, our packaging guide's use has doubled, with over 18,000 unique page views in 2024.
+
+{% include pyos-blockquote.html quote="@leahawasser @pyOpenSci clicking through and eventually found myself looking at 'what is a Python package' and involuntarily performed a standing ovation. bookmarked it as an example of great docs for an incredibly complex subject with many meanings in many different contexts" author="Mastodon Toot" class="highlight" %}
+
+
+Our work has not only made packaging more accessible to more people but also fostered a collaborative and inclusive community of practice where scientists, software maintainers, and Pythonistas hang out, chat about open source and open science challenges, and support each other.
```

Comment:
It's also given pyOpenSci a seat at the table for future Python packaging planning through PyCon's Packaging Summits and Maintainer Summits.

Reply:
fixed locally!